git-notes.md

```diff
@@ -8,3 +8,35 @@ renames, etc.
 ### Adding a file
 
 Use `git add` to tell git that you want it to look after a file for you.
+
+### Save your changes
+
+`git commit`.  Use `-a` to commit changes to all files that have previously
+been added via `git add`. Use `-m` to give a log message.
+
+### See what has changed
+
+`git diff`
+
+### Throw away local changes to a file
+
+`git checkout -- FILENAME`
+
+Warning: do this at your own risk!
+
+`git reset --hard` - do this with all files.
+
+### Make a new branch
+
+Here called "add-documentation". The `-b` tells `git checkout` to create
+the branch.
+
+`git checkout -b add-documentation`
```

Comment:
Hey Terry, you're a bit out of date it seems. Isn't there a command called `git switch`?